File: faq-plus/configapp/Microsoft.Teams.Apps.FAQPlusPlus.Common/Models/TeamsAdaptiveSubmitActionData.cs
// <copyright file="TeamsAdaptiveSubmitActionData.cs" company="Microsoft">
// Copyright (c) Microsoft. All rights reserved.
// </copyright>

namespace Microsoft.Teams.Apps.FAQPlusPlus.Common.Models
{
    using Microsoft.Bot.Schema;
    using Newtonsoft.Json;

    /// <summary>
    /// Defines Teams-specific behavior for an adaptive card submit action.
    /// </summary>
    public class TeamsAdaptiveSubmitActionData
    {
        /// <summary>
        /// Gets or sets the Teams-specific action.
        /// </summary>
        [JsonProperty("msteams")]
        public CardAction MsTeams { get; set; }
    }
}

File: faq-plus/bot/models/teamsAdaptiveSubmitActionData.ts
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.

import { CardAction } from "botframework-schema";

/**
 * Defines Teams-specific behavior for an adaptive card submit action.
 */
export interface TeamsAdaptiveSubmitActionData {
    // Teams-specific action.
    msteams: CardAction
}
